### DISCUSSION TITLE
Question about the newest benchmark results

### DISCUSSION THREAD

**Discussion by 295825725** [janosh/matbench-discovery#100] [Q&A] [answered]
Hi！
I just saw the new benchmark results(https://matbench-discovery.materialsproject.org/) and there were some parts of them that interested me to ask about. I found that the first place was a model called GNoME, so I went to search to learn about this model. Then I found that GNoME is not an open project. They only provide a very simple model framework and many implementation details are not mentioned in the paper as well. However, the latest benchmark table has the results and related details of the GNoME model. That really intrigued me to ask how this is done!
Looking forward to hearing from you!

**Comment by janosh** [comment, marked as answer]
not offensive at all. the matbench discovery test set (we call it [WBM](https://www.nature.com/articles/s41524-020-00481-6)) is public so everybody can test their model on it. this was done internally by DeepMind as we do not have access to GNoME.

in case you're asking if there's any overlap between the GNoME training dataset and our test dataset, the answer is likely yes given that both datasets were generated with element substitution from MP (and in the case of GNoME also OQMD) parent structures. the substitution protocols differ in that GNoME uses a more general partial Wyckoff site substitution which can create new prototypes (see SAPS in the [GNoME paper](https://www.nature.com/articles/s41586-023-06735-9)).

to somewhat control for the likely overlap with WBM GNoME was evaluated twice: once on the complete test set and once only on WBM compositions that do not appear in the GNoME training set. the performance in both cases was similar, which suggests data leakage is not the main reason for GNoME's good performance.

if you're wondering what influence overlapping structure prototypes have, that's more difficult to assess as the SAPS substitution could mean GNoME training data has complete coverage of the prototypes in WBM. [but a recent re-analysis of all open models](https://github.com/janosh/matbench-discovery/pull/75) in matbench discovery that excluded structures from the test set with a matching prototype in MP showed little difference in the model metrics (except for DAF). this suggests prototype overlap is not a significant factor for thermodynamic stability prediction.

see the [relevant section in the paper](https://arxiv.org/abs/2308.14920).
![i4xdsRwF](https://github.com/janosh/matbench-discovery/assets/30958850/d141cfc9-b283-4f25-9682-12933ccc7e23)

**Reply by 295825725** [reply]
Thank you for your comprehensive and detailed reply!
